hue/startup.sh

```diff
@@ -1,12 +1,22 @@
 #!/bin/sh
 
+save_session_data() {
+  echo "Container shutting down, saving session data now..."
+  mariadb-dump --socket=/tmp/maria.sock --databases hue > /tmp/huedb_backup.sql || return
+  mkdir -p /mnt/s3fs/s3-home/.huedb
+  mv /tmp/huedb_backup.sql /mnt/s3fs/s3-home/.huedb/huedb_session_dump.sql
```

Comment:
Didnt you state you had to do an rm on the call we had?

Reply:
yes, but because we are testing the backup command works first I am just moving the successful backup file to the s3 location

Reply:
have now added clearer error reporting and tested in dev for successful and unsuccessful session data dumps